curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-the-environment.english.md

```diff
@@ -10,7 +10,7 @@ forumTopicId: 301566
 <section id='description'>
 For the following challenges, you will be working with a new starter project that is different from the previous one. You can find the new starter project on <a href='https://glitch.com/edit/#!/remix/clone-from-repo?REPO_URL=https://github.com/freeCodeCamp/boilerplate-socketio/'>Glitch</a>, or clone it from <a href='https://github.com/freeCodeCamp/boilerplate-socketio/'>GitHub</a>.
 Add Socket.IO as a dependency and require/instantiate it in your server defined as 'io' with the http server as an argument. <code>const io = require('socket.io')(http);</code>
-The first thing needing to be handled is listening for a new connection from the client. The <dfn>on</dfn> keyword does just that- listen for a specific event. It requires 2 arguments: a string containing the title of the event thats emitted, and a function with which the data is passed though. In the case of our connection listener, we use <em>socket</em> to define the data in the second argument. A socket is an individual client who is connected.
+The first thing needing to be handled is listening for a new connection from the client. The <dfn>on</dfn> keyword does just that- listen for a specific event. It requires 2 arguments: a string containing the title of the event that emitted, and a function with which the data is passed through. In the case of our connection listener, we use <em>socket</em> to define the data in the second argument. A socket is an individual client who is connected.
```

Comment:
Simplify sentences and make it clearer. I replaced a hyphen with an em dash as well.

```suggestion
The first thing to be handle is listening for a new connection from the client. The <dfn>on</dfn> keyword does just that—listen for a specific event. It requires 2 arguments: a string containing the title of the event that's emitted, and a function with which the data is passed through. For our connection listener, we use <em>socket</em> to define the data in the second argument. A socket is an individual client who is connected.
```

Reply:
@erictleung I think you meant "The first thing to be handled is listening for a new connection from the client."

Reply:
@RandellDawson thanks for catching that! I wanted it to be simpler, actually, to just "The first thing to handle..."